### ISSUE TITLE
Td-agent 2

### ISSUE THREAD

**Issue by Seraf** [mmz-srf/puppet-fluentd#35]
Hello,

td-agent 2 has been released. Is there a way to install it with this module actually ?

Announcement : https://groups.google.com/forum/#!topic/fluentd/Qe6W6OCJT-w

Thanks

**Comment by jantman** [comment]
Yeah, this issue makes this repository completely unusable for me, especially since version 1 won't install on CentOS 7.

**Comment by jantman** [comment]
It seems like this *should* be closed by 92f15de via #49
